### DISCUSSION TITLE
Jyers Firmware & Upgraded Touch Panel for E3V2

### DISCUSSION THREAD

**Discussion by jiriteach** [Jyers/Marlin#1286] [Q&A] [answered]
Hi - Does anyone know if this firmware works with the upgraded touch panel for the E3V2 - https://www.creality3dofficial.com/collections/6-for-ender-3-3-pro-5/products/upgraded-touch-panel-screen-display-for-ender-3-pro-ender-3-v2?

Thanks

**Comment by Tuncay-Ayhan** [comment, marked as answer]
It doesn’t. This firmware is build specifically for the combination of a Creality 4.2.2/4.2.7 board with the stock display that comes with the Ender 3v2.

**Reply by jiriteach** [reply]
Thanks for confirming. Am running a 4.2.7 board on an E3V2 but consider upgrading to the touchscreen!
